arcgis-ios-sdk-samples/Geometry/Geodesic operations/GeodesicOperationsViewController.swift

```diff
@@ -0,0 +1,105 @@
+// Copyright 2018 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+
+class GeodesicOperationsViewController: UIViewController, AGSGeoViewTouchDelegate {
+
+    @IBOutlet private var mapView:AGSMapView!
+
+    private var graphicsOverlay = AGSGraphicsOverlay()
+    private var originGraphic: AGSGraphic!
```

Comment:
If you wanted, you could make this (and `destinationGraphic` and `pathGraphic `)  immutable and non-optional by overriding `init(coder:)` and setting them there.

Reply:
I tried modified the code as suggested. Please review it again.